test/project/config/crds/kustomization.yaml

```diff
@@ -0,0 +1,12 @@
+# This file is generated by kubebuilder
+# This kustomization.yaml is not intended to be run by itself,
+# since it depends on service name and namespace that are out of this kustomize package.
+# It should be run by config/default
+resources:
+  - bases/crew_v1_firstmate.yaml
```

Comment:
After thinking a bit more, I thought this may not be a good idea.
`kustomize` restrictions:
- `resources` to explictly to point to a file (__no dir__ is allowed)
- `bases` can point to a dir __with__ a `kustomization.yaml` in it.

Applying the kustomize rules to what you suggested above, the structure will be:
```
test/project/config/crds
├── bases
│   ├── kustomization.yaml
│   ├── creatures_kraken.yaml
│   └── crew_firstmate.yaml
├── kustomization.yaml
├── kustomizeconfig.yaml
└── patches
    ├── creatures_kraken_patch.yaml
    └── crew_firstmate_patch.yaml
```
and `test/project/config/crds/bases/kustomization.yaml` will look like:
```
resources:
- crew_firstmate.yaml  # we still need specify each file here
- creatures_kraken.yaml
```
and `test/project/config/crds/kustomization.yaml` will look like:
```
bases:
- bases/

patches:
- patches/crew_firstmate_patch.yaml
- patches/creatures_kraken_patch.yaml

configurations:
- kustomizeconfig.yaml
```